todo/src/scripts/sortBarView.js

```diff
@@ -0,0 +1,23 @@
+const sortButtonsBlock = document.getElementById('sortButtons');
+let sortDirection;
+
+function initChangeDirectionHandler(currentDirection, callback) {
+    sortDirection = currentDirection;
+    sortButtonsBlock.addEventListener('click', ({target}) => {
+        const isChange = sortButtonClickHandler(target);
+        callback(isChange)
+    })
+}
+
+function sortButtonClickHandler(target) {
+    const targetButton = target.closest('.btn')
+    if (targetButton) {
+        const isChange =  (JSON.parse(targetButton.dataset.sort) === sortDirection);
+        if (isChange) {
+            sortDirection = !sortDirection;
```

Comment:
does it make sense if you not use `sortDirection` further?

Reply:
I added some changes. I use ``` sortDirection``` to avoid rendering when I click  on the button with already selected sort type.
```suggestion
function sortButtonClickHandler(target) {
    const targetButton = target.closest('.btn')
    if (targetButton) {
        const buttonSortValue = JSON.parse(targetButton.dataset.sort);
        const isChange = (buttonSortValue !== sortDirection);
        if (isChange) {
            sortDirection = buttonSortValue;
        }
        return isChange;
    }
}
```

Reply:
just `return buttonSortValue !== sortDirection` should do the same job

Reply:
I tried to use your suggestion, but the sorting direction changes each time when I click on the same button.